SectionGrid.js

```diff
@@ -23,6 +22,7 @@ const SectionGrid = memo(
       onLayout,
       additionalRowStyle: externalRowStyle,
       itemContainerStyle,
+      isInvertedRowItem,
```

Comment:
How about calling this prop `invertedRow`?
We are inverting the row and not the rowItem. Also `is` prefix for booleans are not used by FlatList and SectionList (`horizontal` prop for example) so we can stick to that convention.

Reply:
Thanks for good naming. I will apply it.